src/popup/router/components/Modals/QrCodeReader.vue

```diff
@@ -3,38 +3,59 @@
     v-if="browserReader || !cameraAllowed"
     has-close-button
     class="qr-code-reader"
+    centered
+    from-bottom
     @close="resolve"
+    bodyClass="body-extend"
   >
     <template slot="header">
```

Comment:
For this modal you should not use the #header slot as it's fixed. The purpose is to put the fixed titles there.